### PR TITLE
Email validation synthetics guide

### PR DESCRIPTION
**Pull request by sarina-dd**
### What does this PR do?
Adds the new email validation synthetics guide

### Motivation
Docs request

### Preview link
https://docs-staging.datadoghq.com/sarina/synthetics-email/synthetics/guide/email-validation/

Check preview base path using the URL in details in `preview` status check.

### CODE REVIEW

File: content/en/synthetics/guide/email-validation.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  tag: "Documentation"
+  text: "Learn about steps for browser tests"
+- link: "/synthetics/browser_tests/advanced_options/"
+  tag: "Documentation"
+  text: "Configure advanced options for steps"
+---
+
+## Overview
+
+Classic web application journeys often involve emails being triggered and sent to a users’ mailbox: Email verification after account creation, email sent to reset forgotten passwords, order confirmations, mail confirmation after contact form submission, etc.
```
Comment (sarina-dd) [inline content/en/synthetics/guide/email-validation.md:16]:
```suggestion
Classic web application journeys often involve emails being triggered and sent to a users’ mailbox: email verification after account creation, email sent to reset forgotten passwords, order confirmations, mail confirmation after contact form submission, etc.
```

File: content/en/synthetics/guide/email-validation.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+Making sure your application’s email mechanisms are working as expected is key to ensuring a great user experience on your website.
+
+Datadog Browser Tests allow you to:
+
+- Confirm an email was correctly sent following up on an in-app trigger
+- Check its content
```
Comment (sarina-dd) [inline content/en/synthetics/guide/email-validation.md:23]:
```suggestion
- Check an email's content
```

File: content/en/synthetics/guide/_index.md [hunk 3]
```diff
@@ -5,7 +5,14 @@ private: true
 ---
 
 {{< whatsnext desc="General Guides:" >}}
-    {{< nextlink href="synthetics/guide/uptime-percentage-widget" >}}Website Uptime monitoring with SLO{{< /nextlink >}}
+    {{< nextlink href="synthetics/guide/identify_synthetics_bots/" >}}Identify Synthetics bots{{< /nextlink >}}
+{{< /whatsnext >}}
+
+{{< whatsnext desc="API Tests:" >}}
+    {{< nextlink href="synthetics/guide/uptime-percentage-widget" >}}Website uptime monitoring with SLOs{{< /nextlink >}}
```
Comment (MargotLepizzera) [inline content/en/synthetics/guide/_index.md:12]: Love the split! The `Website uptime monitoring with SLOs` guide showcases an API test but is actually true for both types of tests so I'd rather have it in the general guides section

File: content/en/synthetics/guide/email-validation.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+further_reading:
+- link: "//synthetics/browser_tests/actions"
+  tag: "Documentation"
+  text: "Learn about steps for browser tests"
+- link: "/synthetics/browser_tests/advanced_options/"
+  tag: "Documentation"
+  text: "Configure advanced options for steps"
+---
+
+## Overview
```
Comment (MargotLepizzera) [inline content/en/synthetics/guide/email-validation.md:14]: Can we add an `Overview` section to the other guides for homogeneity or remove it from that one?

File: content/en/synthetics/guide/email-validation.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+- Confirm an email was correctly sent following up on an in-app trigger
+- Check an email's content
+- Click on the links located within the sent emails in order to navigate to other URLs and validate your whole flows, including web and email steps
+
+To perform email validation inside a Datadog Browser test:
+
+## Create an email variable
+
+After creating your browser test, navigate to **Edit Recording**, select **Variables** and then select **Email** from the email variable dropdown list:
```
Comment (MargotLepizzera) [inline content/en/synthetics/guide/email-validation.md:30]:
```suggestion
After filling your Browser test details (start URL, devices, locations, frequency, notification), navigate to **Save Details & Record Test**, select **Variables** and then select **Email** from the email variable dropdown list:
```

File: content/en/synthetics/guide/email-validation.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+{{< img src="synthetics/guide/email-validation/adding-variable.mp4" alt="Create an email variable" video="true"  width="100%">}}
+
+In the above example, an email variable called `EMAIL` is created. The newly created email variable generates a unique mailbox maintained by Datadog at every test execution. This is key to avoid any conflicts among your Browser test runs.
+
+## Record steps
+
+In the upper left corner of the UI, hit the **Start Recording** button and record the steps leading to the email being triggered using your freshly created email variable. Use the hand icon to record these steps.
```
Comment (MargotLepizzera) [inline content/en/synthetics/guide/email-validation.md:38]:
```suggestion
In the upper left corner of the UI, hit the **Start Recording** button and record the steps leading to the email being triggered using your freshly created email variable. Use the hand icon to input variables in the form text inputs.
```

File: content/en/synthetics/guide/email-validation.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+
+## Confirm the email was sent
+
+You can now confirm that an email was sent. Click the **Assertion** button and select the `Test that an email was received` assertion.
+
+{{< img src="synthetics/guide/email-validation/assertion-step.mp4" alt="Add an assertion" video="true"  width="100%">}}
+
+If you want to make sure your email respects specific guidelines for its content, you can add more verifications against its subject and body.
+
+In the above example, the assertion is set to be successful if the email subject is `Welcome to Shopist!` and if its body contains the sentence `Your verification code is` and a verification code matches the `\d{1,6}` regex pattern.
```
Comment (MargotLepizzera) [inline content/en/synthetics/guide/email-validation.md:52]: Link to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions

**Comment by MargotLepizzera** [comment]
This is looking great! I added a few comments, but once these are resolved, it's all good to merge on my end, thanks Sarina!!

**Comment by sarina-dd** [comment]
Resolved all comments from the code review, @MargotLepizzera! 👍

**Comment by MargotLepizzera** [comment]
LGTM, thanks Sarina!

File: content/en/synthetics/guide/uptime-percentage-widget.md [hunk 8]
```diff
@@ -13,11 +13,13 @@ further_reading:
...
 
-Maintaining service level agreements with external or internal customers often requires measuring uptime percentage. This guide shows you how to achieve this using Datadog's [Synthetics][1] and the [SLO widget][2]. It uses `http://example.com/` website as an example.
+## Overview
+
+Maintaining service level agreements with external or internal customers often requires measuring uptime percentage. This guide shows you how to achieve this using Datadog's [Synthetics Monitoring][1] and the [SLO widget][2]. It uses `http://example.com/` website as an example.
 
 ## Synthetics test creation
 
-Create a new [Synthetics API test][3] upon `http://example.com/`:
+Create a new [Synthetics Monitoring API test][3] upon `http://example.com/`:
```
Comment (sarina-dd) [inline content/en/synthetics/guide/uptime-percentage-widget.md:22]:
```suggestion
Create a new [Synthetics API test][3] upon `http://example.com/`:
```

**Pull request merged**